cmd/terramate/cli/cli_test.go

```diff
@@ -160,29 +154,27 @@ func TestListAndRunChangedStack(t *testing.T) {
 	git.CommitAll("stack changed")
 
 	wantList := stack.RelPath() + "\n"
-	assertRunResult(t, cli.run("list", s.BaseDir(), "--changed"),
-		runResult{Stdout: wantList})
+	assertRunResult(t, cli.run("list", "--changed"), runResult{Stdout: wantList})
 
 	cat := test.LookPath(t, "cat")
 	wantRun := fmt.Sprintf(
 		"Running on changed stacks:\n[%s] running %s %s\n%s\n",
-		stack.Path(),
+		stack.RelPath(),
 		cat,
 		mainTfFileName,
 		mainTfContents,
 	)
 
 	assertRunResult(t, cli.run(
 		"run",
-		"--basedir",
-		s.BaseDir(),
 		"--changed",
 		cat,
 		mainTfFileName,
 	), runResult{Stdout: wantRun})
 }
 
 func TestListAndRunChangedStackInAbsolutePath(t *testing.T) {
+	t.SkipNow()
```

Comment:
Is the test supposed to be skipped or is this a leftover from some other change ?

Reply:
not sure what to do with this test because it tests the case where basedir and cwd mismatch, remember? Like `cd /tmp; terramate list /home/iac` in this case it reported absolute paths but now `--chdir <path>` is the new way and we chdir() into the directory first.

Reply:
hmm I see, maybe use t.Skip() with a message making this more clear + we assess this soon